src/main/java/com/example/home_work_8/gameplay/Wall.java

```diff
@@ -0,0 +1,43 @@
+package com.example.home_work_8.gameplay;
+
+public class Wall implements Obstacle {
+
+    private int hight;
+    final String name;
+
+    public Wall(int hight) {
+        this.name = "Wall";
+        this.hight = hight;
+    }
+
+    @Override
+    public int getHight() {
+        return hight;
+    }
+
+    public void setHight(int hight) {
+        this.hight = hight;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public void overcome() {
```

Comment:
this method should accept the parameter `Participant participant` and you should check whether that participant can overcome the distance, if yes then print the appropriate message, if not print another message that the participant failed to overcome the obstacle